src/components/App.js

```diff
@@ -101,6 +119,14 @@ const App = (props) => {
         <IntlProvider locale={locale} messages={allMessages}>
           <AlertDialog />
           <ConfirmDialog confirm={confirm} onConfirm={clearConfirm} />
+          {economicUnitConfig ? (
+            <Contributions
+              contributionKey={ECONOMIC_UNIT_DIALOG_CONTRIBUTION_KEY}
+              open={economicUnitDialogOpen}
+              setEconomicUnitDialogOpen={setEconomicUnitDialogOpen}
+              onLogout={onLogout}
```

Comment:
onLogout imported in App.js and passed as prop to different component. That seems to be the single use of it. Why not import it in that component?

Reply:
It is passed on to the Contributions. The economic unit dialog is implemented in other modules, so you won't be able to import it directly.

Reply:
ok